config/examples/Sovol/SV-01/BigTreeTech SKR Mini E3 3.0/Fixed Probe/Configuration.h

```diff
@@ -0,0 +1,3106 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2021 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * 
+ */
+#pragma once
+
+#define CONFIG_EXAMPLES_DIR "Sovol/SV-01/BigTreeTech SKR Mini E3 2.0"
```

Comment:
This line is added automatically in the `bugfix-2.0.x` branch, so it's not needed/used here.

Reply:
You mean only line 24? or from line 21 until line 24?

Reply:
GitHub must have hiccuped since I only added a comment on the `#define CONFIG_EXAMPLES_DIR "Sovol/SV-01/BigTreeTech SKR Mini E3 2.0"` line.

`CONFIG_EXAMPLES_DIR` is added automatically, so it's not needed on the `import-2.0.x` branch.

Reply:
Ok Thank you

Line deleted